packages/validation/src/diff/LionwebDiff.ts

```diff
@@ -0,0 +1,215 @@
+// TODO The diff is outdated and need work to become ffully functional again.
+
+import { LionWebSyntaxValidator } from "../validators/LionWebSyntaxValidator";
+import {
+    isEqualMetaPointer,
+    LionWebJsonChild,
+    LionWebJsonChunk,
+    LionWebJsonNode,
+    LionWebJsonReference,
+} from "../json/LionWebJson";
+import { NodeUtils } from "../json/NodeUtils";
+import { ChunkUtils } from "../json/ChunkUtils";
+import { ValidationResult } from "../validators/ValidationResult";
+
+export type ChangedType = {
+    path: string;
+    oldValue: any;
+    newValue: any;
+};
+export type AddedOrDeletedType = {
+    path: string;
+    value: any;
+};
+export type MatchedType = {
+    path: string;
+    value: any;
+};
+export type ResultType = {
+    matched: MatchedType[];
+    changed: ChangedType[];
+    added: AddedOrDeletedType[];
+    deleted: AddedOrDeletedType[];
+};
+
+export class LwDiff {
+    errors: string[] = [];
+    lwChecker = new LionWebSyntaxValidator(new ValidationResult());
+
+    constructor() {
+        this.lwChecker.recursive = false;
+    }
+
+    // lwDiff(obj1: any, obj2: any): void {
```

Comment:
Do you have specific plans w.r.t. this commented-out code? If not, I'd prefer to remove it.

Reply:
I had, but not anymore, removed.